src/ducks/trash/Toolbar.jsx

```diff
@@ -1,13 +1,44 @@
 import styles from '../../styles/toolbar'
 
-import React from 'react'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
 import { translate } from '../../lib/I18n'
+import confirm from '../../lib/confirm'
 import classNames from 'classnames'
 
 import Menu, { MenuButton, Item } from 'react-bosonic/lib/Menu'
+import EmptyTrashConfirm from '../../components/EmptyTrashConfirm'
+import Spinner from '../../components/Spinner'
 
-const Toolbar = ({ t, disabled, onSelectItemsClick }) => (
+import { emptyTrash } from './actions'
+
+class DeleteButton extends Component {
+  state = {
+    working: false
+  };
```

Comment:
The extra ';' is required by the standard for class properties, this sucks but it's necessary